modules/preferences/main/index.coffee

```diff
@@ -230,7 +230,13 @@ class Preferences extends hx.EventEmitter
         -(moment.tz.zone(timezone).offset(timestamp) / 60)
       @timezone guessedMomentTimezone
     else
-      @timezone 'UTC+00:00'
+      zeroPad = hx.format.zeroPad(2)
+      offset = (new Date()).getTimezoneOffset()
+      modifier = if offset > 1 then '-' else '+'
+      absOffset = Math.abs(offset)
+      minutes = absOffset % 60
+      hours = (absOffset - minutes) / 60
+      @timezone "UTC#{modifier}#{zeroPad(hours)}:#{zeroPad(minutes)}"
```

Comment:
This feels like the construction of the string should be wrapped up in a separate function, and should have some tests to test that function